install/src/main/java/org/jetbrains/bsp/bazel/install/cli/CliOptionsProvider.kt

```diff
@@ -91,11 +91,19 @@ class CliOptionsProvider(private val args: Array<String>) {
             .hasArg()
             .argName("path")
             .desc(
-                "Add java path to the generated project view file, you can read more about it here: " +
-                        "https://github.com/JetBrains/bazel-bsp/tree/master/executioncontext/projectview#java_path."
+            "Add java path to the generated project view file, you can read more about it here: " +
+                    "https://github.com/JetBrains/bazel-bsp/tree/master/executioncontext/projectview#java_path."
             )
             .build()
         cliParserOptions.addOption(javaPathOption)
+
+        val manualTargetsOption = Option.builder(BUILD_MANUAL_TARGETS_OPT)
+            .longOpt("build_manual_targets")
+            .desc(
+            "Add manual tag to omit wild cards with this tag. Compiling, testing and running can be done by explicitly mentioning.")
```

Comment:
it doesnt explain what the flag does - also u should update https://github.com/JetBrains/bazel-bsp/tree/master/executioncontext/projectview and provide a link here